### ISSUE TITLE
Copy and paste error in P02_C02

### ISSUE THREAD

**Issue by KellenSunderland** [zackchase/mxnet-the-straight-dope#68]
Looks like the gluon version of linear regression still has the training steps from the hand-rolled example of linear regression.

> Generate predictions (yhat) and the loss (loss) by executing a forward pass through the network.
> Calculate gradients by making a backwards pass through the network (loss.backward()).
> Update the model parameters by invoking our SGD optimizer.

**Comment by zackchase** [comment]
Thanks! Yeah that one was a bit naked anyway still. I added some copy and pushed. You're becoming indispensable :)
